src/components/diagrams/singleLineDiagram/single-line-diagram-pane.js

```diff
@@ -224,7 +225,7 @@ export function SingleLineDiagramPane({
     const classes = useStyles();
 
     const handleUpdateSwitchState = useCallback(
-        (breakerId, open, switchElement) => {
```

Comment:
keep it even if not used, it's pass throught SingleLineDiagram:onBreakClick.
Or clean this in another PR.